ui/desktop/app/components/settings-card/client-agent/index.hbs

```diff
@@ -0,0 +1,72 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if this.isClientAgentActive}}
+  <SettingsCard
+    @header={{t 'settings.client-agent.title'}}
+    @icon='network-alt'
+    @description={{t 'settings.client-agent.description'}}
+  >
+    <:state>
+      {{#if this.isClientAgentStatusRunning}}
+        <Hds::Badge
+          @text={{t 'states.running'}}
+          @icon='running'
+          @color='success'
+        />
+      {{else}}
+        <Hds::Badge
+          @text={{t 'states.paused'}}
+          @icon='pause-circle'
+          @color='neutral'
+        />
+      {{/if}}
+    </:state>
+    <:body>
+      <div>
+        <Hds::Text::Display>
+          {{t 'settings.version'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @weight='medium'>
+          {{@model.clientAgentStatus.version}}
+        </Hds::Text::Body>
+      </div>
+      {{#if @model.clientAgentStatus.errors}}
+        <Hds::Alert @type='inline' @color='warning' as |A|>
+          <A.Title>{{t 'settings.alerts.client-agent'}}</A.Title>
+          <A.Description>
+            {{#each @model.clientAgentStatus.errors as |error|}}
+              <Hds::Text::Body @tag='p'>
```

Comment:
non-blocking: why is the `hds::text` component needed here since the `hds::alert` component automatically applies css tokens? Is it used mainly to help format the errors perhaps ?

Reply:
Well the main thing is there can be multiple errors so I could either do multiple `Description` fields which didn't seem right or I could have multiple text components inside the description for each error. It's to format the errors and to semantically have them be their own html element